### PR TITLE
Refactor the code styles

### PR DESCRIPTION
**Pull request by NipunaMarcus**

### CODE REVIEW

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddDeleteDashboardTest.java [hunk 1]
```diff
@@ -68,22 +69,20 @@ public void setUp() throws Exception {
...
         driver.findElement(By.id("ues-dashboard-title")).clear();
         driver.findElement(By.id("ues-dashboard-title")).sendKeys(dashboardTitle);
         driver.findElement(By.id("ues-dashboard-description")).clear();
         driver.findElement(By.id("ues-dashboard-description")).sendKeys(DASHBOARD_DESCRIPTION);
         driver.findElement(By.id("ues-dashboard-create")).click();
-        driver.findElement(By.id("single-column")).click();
-        driver.findElement(By.cssSelector("a.navbar-brand.ues-tiles-menu-toggle")).click();
-        driver.findElement(By.cssSelector("i.fw.fw-dashboard")).click();
+        driver.findElement(By.cssSelector("a[data-id='single-column']")).click();
+        redirectToLocation("portal", "dashboards");
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddDeleteDashboardTest.java]: Can this portal and dashboard defined as constants?

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java [hunk 2]
```diff
@@ -145,16 +141,16 @@ public void testMaximizeGadgetInView() throws Exception {
 
         Object txt = driver.executeScript(
                 "var iframe = $(\"iframe[title='USA Map']\")[0];" +
-                "var innerDoc = iframe.contentDocument || iframe.contentWindow.document;" +
-                "return innerDoc.getElementById('defaultViewLabel').textContent;"
+                        "var innerDoc = iframe.contentDocument || (iframe.contentWindow && iframe.contentWindow.document);" +
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java]: formatting issue.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java [hunk 3]
```diff
@@ -145,16 +141,16 @@ public void testMaximizeGadgetInView() throws Exception {
...
+                        "return innerDoc.getElementById('defaultViewLabel').textContent;"
         );
 
         assertEquals("USA MAP (This is default view)", txt.toString());
 
         String showToolbarScript =
                 "for(i = 0; i < document.getElementsByClassName('ues-component-toolbar').length; i++) {" +
-                "    document.getElementsByClassName('ues-component-toolbar')[i].style.display = 'inline';" +
-                "}";
+                        "    document.getElementsByClassName('ues-component-toolbar')[i].style.display = 'inline';" +
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java]: Formatting issue.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java [hunk 4]
```diff
@@ -165,8 +161,8 @@ public void testMaximizeGadgetInView() throws Exception {
         //maximized Window view
         Object txtMax = driver.executeScript(
                 "var iframe = $(\"iFrame[title='USA Map']\")[0];" +
-                "var innerDoc = iframe.contentDocument || iframe.contentWindow.document;" +
-                "return innerDoc.getElementById('fullViewLabel').textContent;"
+                        "var innerDoc = iframe.contentDocument || (iframe.contentWindow && iframe.contentWindow.document);" +
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java]: Formatting issue.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java [hunk 5]
```diff
@@ -62,24 +63,22 @@ public void testAddEditPageDashboardNew() throws Exception {
...
+        driver.executeScript("$('[name=title]').change();");
         driver.findElement(By.cssSelector("[name=id]")).clear();
         driver.findElement(By.cssSelector("[name=id]")).sendKeys(DASHBOARD_URL);
-        driver.findElement(By.cssSelector("h4.ues-page-title")).click();
+        driver.executeScript("$('[name=id]').change();");
 
-        assertEquals(driver.findElement(By.cssSelector("h4.ues-page-title")).getText(), DASHBOARD_PAGENAME,
+        assertEquals(driver.findElement(By.cssSelector("h4.ues-page-title")).
+                        findElement(By.cssSelector("span.page-title")).getText(), DASHBOARD_PAGENAME,
                 "error occurred while edit the new page name");
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java]: Formatting issue, code indentation.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java [hunk 6]
```diff
@@ -93,18 +92,18 @@ public void testAddEditPageDashboardNew() throws Exception {
             "testAddEditPageDashboardNew")
     public void testLandingCheckBox() throws Exception {
         DSWebDriver driver = getDriver();
+
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java]: Remove this blank line.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java [hunk 7]
```diff
@@ -31,6 +31,7 @@
 import java.util.List;
 
 import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
 public class AddPageToDashboard extends DSUIIntegrationTest {
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java]: Add class comments.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java [hunk 8]
```diff
@@ -113,17 +112,13 @@ public void testLandingCheckBox() throws Exception {
     public void testDeletePageDashboardNew() throws Exception {
         DSWebDriver driver = getDriver();
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java]: Remove unnecessary blank lines throughout the code.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java [hunk 9]
```diff
@@ -27,6 +27,8 @@
 import org.wso2.carbon.automation.engine.context.TestUserMode;
 import org.wso2.ds.ui.integration.util.DSUIIntegrationTest;
 
+import java.util.concurrent.TimeUnit;
+
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java]: Remove this blank line.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java [hunk 10]
```diff
@@ -61,95 +72,42 @@ public void setUp() throws Exception {
...
-                "draggableOffset = draggableElement.offset()," +
-                "dx = droppableOffset.left - draggableOffset.left," +
-                "dy = droppableOffset.top - draggableOffset.top;" +
-                "draggableElement.simulate('drag', {" +
-                "dx: dx," +
-                "dy: dy" +
-                "});");
-
-        String defaultDesignerModeGadgetActual = getDriver().findElement(By.cssSelector("iframe")).getAttribute("title");
+        assertEquals(GADGET_2, getAttributeValue("iframe", "title"));
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java]: Remove these blank lines..

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java [hunk 11]
```diff
@@ -268,17 +165,18 @@ public void testAnonDashboardPages() throws Exception {
...
-        String defaultGadgetActual = getDriver().findElement(By.cssSelector("iframe")).getAttribute("title");
-        getDriver().get(getBaseUrl() + "/portal/dashboards/" + dashboardTitle + "/landing?isAnonView=true");
-        boolean isLinkForNonAnonPageNotExist = getDriver().findElements(By.cssSelector("a[href='" + getBaseUrl() +
-                "/portal/dashboards/" + dashboardTitle + "/page0?isAnonView=true']")).size() <= 0;
 
-        assertEquals(defaultGadgetExpected, defaultGadgetActual);
         assertTrue(isToggleButtonHidden, "Anonymous toggle button is not hidden");
+        assertEquals(GADGET_3, getAttributeValue("iframe", "title"));
+
+        redirectToLocation("portal", "dashboards/" + dashboardTitle + "/landing?isAnonView=true");
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java]: Export the commonly used URIs to a constant class, and reuse in all the locations.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java [hunk 12]
```diff
@@ -288,24 +186,33 @@ public void testAnonDashboardPageRemove() throws Exception {
...
     }
 
+    /**
+     * Get the value of given attribute of given element
+     *
+     * @param element       name of the element
+     * @param attributeName name of the attribute
+     * @return {String}
+     */
+    public String getAttributeValue(String element, String attributeName) throws Exception {
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java]: Why it's public? If this method needs to be used within this class then it should be a private method.

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddDeleteDashboardTest.java (hunk 1 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddDeleteDashboardTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java (hunk 2 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java (hunk 3 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java (hunk 4 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddGadgetToDashboardTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java (hunk 5 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java (hunk 6 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java (hunk 7 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java (hunk 8 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/AddPageToDashboard.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java (hunk 9 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java (hunk 10 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java (hunk 11 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java (hunk 12 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/DashboardAnonViewTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

**Pull request merged**